lib/widgets/currency_converter_dialog.dart

```diff
@@ -123,6 +131,11 @@ class CurrencyConverterDialogState extends State<CurrencyConverterDialog>
               .exchangeRate;
           _updateExchangeLabel(exchangeRate);
 
+          List<FiatConversion> usersFiatConversions = List.from(account
```

Comment:
This patterns repeats itself and I think the account model can encapsulate that. 
The Account bloc listens for user changes already and push a new AccountModel with each change so we can add a property on the AccountModel preferredFiatConversions that is already the list you need here.
We will save all the previous line of codes as well.
Also a better way is to use the StreamBuilder instead of this initialisation code.

Reply:
[Done](https://github.com/breez/breezmobile/pull/394/commits/b735bf0282c1cdff6ac9266c28c9d2c6cd23e13c)